src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTestSwitchExpression.java

```diff
@@ -0,0 +1,120 @@
+/*
+RightCurly
+option = ALONE
+tokens = LITERAL_SWITCH, LITERAL_IF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.blocks.rightcurly;
```

Comment:
Add comment `//non-compiled with javac: Compilable with Java14` above the package exactly as is, for anything that requires Java 14.